whois/table_whois_domain.go

```diff
@@ -139,3 +154,115 @@ func containsString(items []string, target string) bool {
 	}
 	return false
 }
+
+func rdapToWhoisDomain(passedDomain string, r *rdap.Domain) whoisparser.WhoisInfo {
```

Comment:
@lyda Based on some of the conversions/field matches I see in this function, do you think it'd be beneficial to have a separate `rdap_domain` table in this plugin? I think one plugin is better than 2 overall, but RDAP records seem to have enough differences that warrant a separate table with RDAP specific column names.